command/peers_status.go

```diff
@@ -20,8 +35,9 @@ func (p *PeersStatus) GetHelperText() string {
 // Help implements the cli.PeersStatus interface
 func (p *PeersStatus) Help() string {
 	p.Meta.DefineFlags()
+	p.DefineFlags()
 
-	usage := "peers status PEER_ID"
+	usage := "peers status --libp2p-node-id PEER_ID"
```

Comment:
Kind of a nitpick, but I'm trying to think of a different way we can name this flag.
**libp2p-node-id** just seems a bit long, especially for a command that just returns basic peer data.

What you you think about just **peer-id**?
@Kourin1996 @lazartravica

Reply:
I think **peer-id** sounds ok. Can understand fully enough. It's better to change help message also if you change to **peer-id**.

Reply:
My intention was to add more context to the flag itself, but I agree, the command is often used, so the flag should be shorter. Will modify.